cfme/test_framework/sprout/client.py

```diff
@@ -122,3 +122,6 @@ def provision_appliances(
 
     def destroy_pool(self, pool_id):
         self.call_method('destroy_pool', id=pool_id)
+
+    def all_availiable_versions(self):
```

Comment:
typo.

Reply:
And by the way, sprout client already has this behaviour, you are just overriding what is the exactly same behaviour :)